Resources/Prototypes/Recipes/Cooking/meal_recipes.yml

```diff
@@ -483,6 +532,7 @@
   result: WeaponBaguette
   secretRecipe: true
   time: 15
+  group: Breads
```

Comment:
Since this can only be cooked in the DonkCo microwave, it might be worth putting this in a special group or something. Players might be confused why they can't cook it otherwise.